packages/website/plugins/generated-rule-docs/RuleDocsPage.ts

```diff
@@ -1,8 +1,9 @@
 import type { ESLintPluginRuleModule } from '@typescript-eslint/eslint-plugin/use-at-your-own-risk/rules';
+import { fromMarkdown } from 'mdast-util-from-markdown';
```

Comment:
I accidentally coincidentally found in this PR that #5393 is no longer blocked! Hooray! Will split that out into a separate PR when I get a chance.

Reply:
Merged.